clarify config generation and tool (#426)
commerce-docs/microsite-commerce-storefront: src/content/docs/get-started/index.mdx

### Link repo to commerce data

> To link your storefront repo to your commerce, you can duplicate the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) from your storefront repo and rename it to `config.json`. Use the `demo-config.json` file as a starting point for your site's configuration. You can also use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site.

Reviewer: ```suggestion
To link your storefront repo to your Commerce backend:
- Duplicate the [demo-config.json](https://github.com/hlxsites/aem-boilerplate-commerce/blob/main/demo-config.json) from your storefront repo and rename it to `config.json`. The `demo-config.json` is only meant as a starting point for your site's configuration.
- Alternatively, use the [config generator tool](https://da.live/app/adobe-commerce/storefront-tools/tools/config-generator/config-generator) to automatically generate a `config.json` file for your site.
```

> Make sure to validate the configuration, and then push the `config.json` file either to your repo or to `publicConfig` if you are using [Config Service](https://www.aem.live/docs/config-service-setup)).

Reviewer: ```suggestion
Make sure to validate the configuration, then push the `config.json` file to your repo or to `publicConfig` if you are using [Config Service](https://www.aem.live/docs/config-service-setup).
```
